### ISSUE TITLE
typo in opensta.pdf

### ISSUE THREAD

**Issue by Ravenslofty** [parallaxsw/OpenSTA#55]
> A useful idiom to find the driver pin for a net is the following.
> `get_pins -of_objects [get_net net_name] -filter direction == output`

This should be `get_pins -of_objects [get_net net_name] -filter "direction == output"`, I think. Using the line directly results in obscure errors like
```
Warning: fast_characterise.tcl line 12, patterns argument not supported with -of_objects.
Error: fast_characterise.tcl line 12, unsupported pin -filter expression.
```

**Comment by jjcherry56** [comment]
Correct. The other option is to remove the spaces in the filter expression so it is one tcl token.
I will update the docs.

**Comment by jjcherry56** [comment]
fixed by 9179f808 doc
